ja/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md

```diff
@@ -0,0 +1,245 @@
+
+## その他の注目すべき 2.6 からの変更点
+
+* いくつかの標準ライブラリがアップデートされました。
+  * Bundler 2.1.0.pre.3 ([History](https://github.com/bundler/bundler/blob/2-1-stable/CHANGELOG.md#210pre3-november-8-2019))
+  * RubyGems 3.1.0.pre.3 ([History](https://github.com/rubygems/rubygems/blob/3.1/History.txt))
+  * CSV 3.1.2 ([NEWS](https://github.com/ruby/csv/blob/v3.1.2/NEWS.md))
+  * Racc 1.4.15
+  * REXML 3.2.3 ([NEWS](https://github.com/ruby/rexml/blob/v3.2.3/NEWS.md))
+  * RSS 0.2.8 ([NEWS](https://github.com/ruby/rss/blob/v0.2.8/NEWS.md))
+  * StringScanner 1.0.3
+
+* 以下のライブラリが新たに default gems になりました
+  * 以下のライブラリは rubygems.org からアップデート可能です。
+    * benchmarck
+    * cgi
+    * delegate
+    * getoptlong
+    * net-pop
+    * net-smtp
+    * open3
+    * pstore
+    * singleton
+  * 以下のライブラリはまだ rubygems.org から取得できません。
+    * monitor
+    * observer
+    * timeout
+    * tracer
+    * uri
+    * yaml
+
+* ブロックを渡すメソッド呼び出し中の、ブロックを伴わない `Proc.new` と `proc` が警告されるようになりました。
+
+* ブロックを渡すメソッド呼び出し中の、ブロックを伴わない `lambda` はエラーとなるようになりました。
+
+* Unicode および Emoji のバージョンが 11.0.0 から 12.0.0 になりました。[[Feature #15321]](https://bugs.ruby-lang.org/issues/15321)
+
+* Unicode のバージョンが 12.1.0 となり、新元号「令和」を表す合字 U+32FF がサポートされました。[[Feature #15195]](https://bugs.ruby-lang.org/issues/15195)
+
+* `Date.jisx0301`, `Date#jisx0301`, および `Date.parse` で新元号に対応しました。[[Feature #15742]](https://bugs.ruby-lang.org/issues/15742)
+
+* Ruby のビルドに C99 に対応したコンパイラが必要になりました。[[Misc #15347]](https://bugs.ruby-lang.org/issues/15347)
+  * 本件についての詳細: <https://bugs.ruby-lang.org/projects/ruby-trunk/wiki/C99>
+
+その他詳細については、[NEWS](https://github.com/ruby/ruby/blob/v2_7_0_preview3/NEWS) ファイルまたは[コミットログ](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview3)を参照してください。
+
+なお、こうした変更により、Ruby 2.6.0 以降では [3895 個のファイルに変更が加えられ、213426 行の追加と 96934 行の削除が行われました](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview3) !
+
+Ruby 2.7 で楽しいプログラミングを！
+
+## Download
+
+* <https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.0-preview3.tar.bz2>
+
+      SIZE:   14628168 bytes
+      SHA1:   849a96041202ad3beea82e1069694740d907d583
+      SHA256: 09de8d3a014904142b7bc111195ba8094f6eeb59eb529f7bb0512d8b0fc06f5d
+      SHA512: 661eaff1830546cea9e36f1f5a3cb591d792a17ad18d2c00bd097d53757bf321f0d192ddb3c8fdc75c28a0ecbef587e433ad1b345cdeeb4aa6a4744f7e828fbb
+* <https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.0-preview3.tar.gz>
+
+      SIZE:   16715144 bytes
+      SHA1:   a90300e13b4c8778c798e60b7d782f9808cfe492
+      SHA256: 54300275a17a0367bab91ad54b51faf3d45955041f0bf69a4612fe7b0179f4f2
+      SHA512: 9083525de830a4b2e8685681dfaa3bce14cc1ed7360f5540211f6aa97967e316983554596adf442dda52041245e31ec42bffd1982d28ad5e5c4c230be3fa7d62
+* <https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.0-preview3.tar.xz>
+
+      SIZE:   11926312 bytes
+      SHA1:   b40a6c8c11a0ba7c0d9f1548358da7c662673852
+      SHA256: aaf3d78e2d7b3d42c668f9f619a016507771552e207f7729b3feba19b13c591a
```

Comment:
I get another checksum when downloading https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.0-preview3.tar.xz

```
$ wget https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.0-preview3.tar.xz
--2019-11-23 23:43:16--  https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.0-preview3.tar.xz
Resolving cache.ruby-lang.org (cache.ruby-lang.org)... 151.101.85.178, 2a04:4e42:14::434
Connecting to cache.ruby-lang.org (cache.ruby-lang.org)|151.101.85.178|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 11923988 (11M) [application/x-tar]
Saving to: ‘ruby-2.7.0-preview3.tar.xz’

ruby-2.7.0-preview3 100%[===================>]  11.37M  14.9MB/s    in 0.8s

2019-11-23 23:43:17 (14.9 MB/s) - ‘ruby-2.7.0-preview3.tar.xz’ saved [11923988/11923988]
```
```
$ shasum -a 256 ruby-2.7.0-preview3.tar.xz
ad9d61e55ac224e3447a762e001965839846f9658f87a0e792840887cfe61b8c  ruby-2.7.0-preview3.tar.xz
```

Reply:
Looks like they are all wrong? This is what I'm getting

```
$ shasum -a 256 ruby-2.7.0-preview3.*
df2ddee659873e6fc30a8590ecffa49cf3a4ef81fa922b0d09f821b69ee88bc3  ruby-2.7.0-preview3.tar.bz2
9baa1f5096ebc2a0923df628d7dc7105da3789c1bf8b873469d9010249736b00  ruby-2.7.0-preview3.tar.gz
ad9d61e55ac224e3447a762e001965839846f9658f87a0e792840887cfe61b8c  ruby-2.7.0-preview3.tar.xz
2bc95f67f271b6a41fc3dd40536705b4a7974df8a2fa33ff0758a60822291781  ruby-2.7.0-preview3.zip
```

Reply:
Confirmed.  I think it is caused by wrong copy-and-paste.  I've created #2284